### PR TITLE
formulaire 1. tuto écrit + code complet

### PR DESCRIPTION
**Pull request by GtAntoine**
Petites modifications des fichiers package.json et src/application.js à reporter sur la branche tutorial.

### CODE REVIEW

File: src/views/user/card.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const cardStyle= {
+  flex: 1,
+  minWidth: '300px',
+  maxWidth: '300px',
+  marginTop: '7px',
+  marginRight: '20px',
+  marginBottom: '20px'
+};
+
+const Card = ({title,description, route, destination }) => {
```
Comment (TomGallon) [inline src/views/user/card.js]: why don't you use the Panel component ?

https://github.com/get-focus/focus-components/tree/develop/src/panel

Comment (TomGallon) [reply src/views/user/card.js]: if it is useless, please remove this component.

**Review by TomGallon — CHANGES_REQUESTED** [review]

File: src/services/index.js [hunk 2]
```diff
@@ -1 +1,16 @@
-// déclarez ici les appels aux API de services métier
+import focusFetch from 'focus-application/fetch/fetch-proxy'
+
+export const loadUser = async ({id}) => {
```
Comment (Ephrame) [inline src/services/index.js:3]: For the loadUser you have to create a webService specific, like this, the "then" is not necessary

File: src/views/user/user-form.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+};
+
+const formConfig = {
+    formKey: 'userForm',
+    entityPathArray: ['user'],
+    loadAction: loadUserAction,
+    saveAction: saveUserAction
+};
+
+const ConnectedUserForm = compose(
```
Comment (Ephrame) [inline src/views/user/user-form.js:51]: When there is just one 'metadata' it's not necessary to declare it in each fieldFor

**Review by Ephrame — COMMENTED** [review]
Can you put some comments to explain the line in the code directly ?

**Comment by Ephrame** [comment]
There is no changement about the service ?

File: src/views/user/user-form.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+import {connect as connectToMetadata} from 'focus-graph/behaviours/metadata';
+import {connect as connectToFieldHelpers} from 'focus-graph/behaviours/field';
+import {connect as connectToMasterData} from 'focus-graph/behaviours/master-data';
+import Panel from 'focus-components/panel';
+
+import {loadUserAction, saveUserAction} from '../../actions/user-actions';
+
+const User = ({fieldFor, selectFor, ...otherProps}) => (
+    <Panel title='User' {...otherProps}>
+        {fieldFor('uuid', {entityPath: 'user'})}
```
Comment (Ephrame) [inline src/views/user/user-form.js]: Why do you keep the entityPath ?

File: src/services/index.js [hunk 5]
```diff
@@ -1 +1,19 @@
...
+    return focusFetch({url: `http://localhost:9999/x/users/${id}`, method: 'GET'}).then((data) => {
+        return {
+            ...data.user,
+            __Focus__updateRequestStatus: data.__Focus__updateRequestStatus
+        };
+    });
+};
```
Comment (TomGallon) [inline src/services/index.js:10]: Please remove the then. It is useless.

**Review by TomGallon — CHANGES_REQUESTED** [review]

File: src/services/index.js (hunk 2 above)
Comment (GtAntoine) [reply src/services/index.js:3]: Not sure how to write it without the then

**Pull request merged**